src/Microsoft.Extensions.Localization/Internal/IResourceStringManager.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+
+namespace Microsoft.Extensions.Localization.Internal
+{
+    public interface IResourceStringManager
+    {
+        string GetResourceName(CultureInfo culture);
+
+        IList<string> GetAllResourceStrings(CultureInfo culture);
```

Comment:
Your interface here exposes both the `resource name` concept and the `all strings` concept. Is there a way we could  relayer this code to get rid of `GetResourceName`? This isn't critical as a temporary/internal thing, this is already an improvement.